### PR TITLE
[WIP] Yd jewel mailmap updates

### PR DESCRIPTION
**Pull request by dupont-y**
This a work in progress. Not totally covered all changes for jewel.

Loic, you can start to review  some of the changes at your will. I hope to be totally synced tomorrow.

### CODE REVIEW

File: .mailmap [hunk 1]
```diff
@@ -77,6 +77,7 @@ Florent Manens <florent@beezim.fr> Florent Manens <florent@manens.org>
 Florian Coste <fcoste21@gmail.com> nairolf21
 Florian Marsylle <florian.marsylle@hotmail.fr> <floleblanc@hotmail.fr>
 François Lafont <flafdivers@free.fr> <francois.lafont@crdp.ac-versailles.fr>
+François Lafont <flafdivers@free.fr> lafont <francois.lafont@ac-versailles.fr>
```
Comment (theanalyst) [inline .mailmap]: Think you can drop the `lafont` here .. ie 
```François Lafont <flafdivers@free.fr> <francois.lafont@ac-versailles.fr>``` should work

File: .mailmap [hunk 2]
```diff
@@ -321,4 +331,5 @@ Zhiqiang Wang <zhiqiang.wang@intel.com> Signed-off-by: Zhiqiang Wang <wonzhq@hot
 Zhiqiang Wang <zhiqiang.wang@intel.com> Wang, Zhiqiang <zhiqiang.wang@intel.com>
 Zhiqiang Wang <zhiqiang.wang@intel.com> Zhiqiang Wang <wonzhq@hotmail.com>
 Zhiqiang Wang <zhiqiang.wang@intel.com> Zhiqiang Wang <zhiqiang.wang@intel.com>
-Zhi Zhang <zhangz.david@outlook.com> <zhangzhi@localhost.localdomain>
+Zhi Zhang <willzzhang@tencent.com> Zhi Zhang <zhangz.david@outlook.com>
+Zhi Zhang <willzzhang@tencent.com> Zhi Zhang <willzzhang@tencent.com>
```
Comment (theanalyst) [inline .mailmap]: Newline at end of the file

File: .mailmap [hunk 3]
```diff
@@ -245,6 +249,7 @@ Shanggao Qiu <qiushanggao@qq.com> qiushanggao
 Shawn Chen <cxwshawn@gmail.com> <cxwshawn@gmail.com>
 Shu, Xinxin <xinxin.shu@intel.com> xinxin shu <xinxin.shu@intel.com>
 Shu, Xinxin <xinxin.shu@intel.com> xinxinsh <xinxin.shu@intel.com>
+Song Baisen <song.baisen@zte.com.cn>
```
Comment (theanalyst) [inline .mailmap]: Does an entry like this change the output or so.. As far the author has used only one email, I guess this entry could be dropped

**Comment by theanalyst** [comment]
Just a few comments looks good otherwise

File: .mailmap (hunk 2 above)
Comment (dupont-y) [inline .mailmap]: OK, will correct & rebase

File: .mailmap (hunk 3 above)
Comment (dupont-y) [inline .mailmap]: Well, that's for normalisation : original entry is like that : songbaisen <song.baisen@zte.com.cn>
I assumed Song Baisen is better here.

File: .mailmap (hunk 1 above)
Comment (dupont-y) [inline .mailmap]: That's right. There are probably others occurences of this unnecessary "name + email" in this file...

File: .mailmap (hunk 3 above)
Comment (dupont-y) [inline .mailmap]: Le 04/02/16 14:58, Abhishek L a écrit :
>
> In .mailmap <https://github.com/ceph/ceph/pull/7517#discussion_r51874918>:
>
> > @@ -245,6 +249,7 @@ Shanggao Qiu <qiushanggao@qq.com> qiushanggao
> >  Shawn Chen <cxwshawn@gmail.com> <cxwshawn@gmail.com>
> >  Shu, Xinxin <xinxin.shu@intel.com> xinxin shu <xinxin.shu@intel.com>
> >  Shu, Xinxin <xinxin.shu@intel.com> xinxinsh <xinxin.shu@intel.com>
> > +Song Baisen <song.baisen@zte.com.cn>
>
> Does an entry like this change the output or so.. As far the author 
> has used only one email, I guess this entry could be dropped
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/ceph/ceph/pull/7517/files#r51874918>.
>
BTW, I was reading https://git-scm.com/docs/git-shortlog , .mailmap 
section, and I have the feeling our .mailmap is over complicated.

For example, in my yesterday pull request,  patch 
64c0e25673d1e11301713b0f51ad136ae9802990 I had to made a fix for sage 
typo in his name (sage Weli) but email is good.

This one is not needed, a single Sage Weil <sweil@redhat.com> should be 
sufficient, all variant could be eliminated.
I've made some of these errors in the past, and I'm quite sure we can 
somewhat simplify this file.

I'll try this if you're ok with that.

**Comment by dupont-y** [comment]
I have done extensive cleanup of .mailmap.
I'm rebasing all other patches.
I'm preparing another pull request.

This one will be obsoleted soon.

**Comment by dupont-y** [comment]
new one is https://github.com/ceph/ceph/pull/7528

this one is not complete either.

**Pull request closed without merging**